### DISCUSSION TITLE
Scale a project out to multiple instances

### DISCUSSION THREAD

**Discussion by ejsmith** [microsoft/aspire#1488] [Q&A] [answered]
I want to run a project with AddProject, but I want to scale it to multiple instances in order to more closely emulate my application in production and make sure that things work correctly when there is more than one instance of my service running.

**Comment by davidfowl** [comment, marked as answer]
Call WithReplicas(number)

**Reply by ejsmith** [reply]
That is so bad ass! I was looking for it, but I didn't see it. Thank you!
